### DISCUSSION TITLE
Video on OpenMPTCPRouter in virtual testlab on Proxmox

### DISCUSSION THREAD

**Discussion by onemarcfifty** [Ysurac/openmptcprouter#1720] [Show and tell]
Hi chaps, I have just created a video about running OpenMPTCPRouter in a virtual Testlab (on Proxmox) - the goal was to

- Test OpenMPTCPRouter under real-life conditions (i.e. "bad" lines, lines with different bw/latency etc.)
- Test OMR without having to buy 4G/LTE lines
- Build a versatile network test lab in Proxmox

The video is [here on my Youtube channel](https://youtu.be/mYYoIDCWszo) - enjoy

@Ysurac - if you want to you can link it somewhere (maybe in the tutorials section?)

**Comment by chop249** [comment]
> Hi chaps, I have just created a video about running OpenMPTCPRouter in a virtual Testlab (on Proxmox) - the goal was to
> 
> * Test OpenMPTCPRouter under real-life conditions (i.e. "bad" lines, lines with different bw/latency etc.)
> * Test OMR without having to buy 4G/LTE lines
> * Build a versatile network test lab in Proxmox
> 
> The video is [here on my Youtube channel](https://youtu.be/mYYoIDCWszo) - enjoy
> 
> @Ysurac - if you want to you can link it somewhere (maybe in the tutorials section?)

@onemarcfifty Did you run OMRVPS on ProxMox? I'm just looking into ProxMox now to hopefully combine OMRVPS, Plex, Pi-Hole and Unbound into 1 machine instead of 3. It'd be awesome if I could also move my Webmin/Virtualmin website server to that same machine.

**Reply by onemarcfifty** [reply]
Yes - I had everything run on Proxmox. Just remember - this is for TESTING, i.e. you will not get better bandwidth in the real world with this ;-)
